charts/monasca/templates/role-binding.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.rbac.enabled }}
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+metadata:
+  name: "monasca-role-binding"
```

Comment:
if I deploy this twice what is the behavior? like deploying chart into the same cluster or namespace twice